charts/datadog/templates/agent-cilium-network-policy.yaml

```diff
@@ -35,6 +35,12 @@ specs:
           - ports:
               - port: "123"
                 protocol: UDP
+      - toCIDR:
+          - 169.254.169.123/32
```

Comment:
I would prefer to have a list on ip range in the `values.yaml` to easily update/add other ip range

Reply:
Also quick questions (because I'n not an expert on cilium network policy)
* what is the behaviour of the rule if `toCIDR` is empty? does it mean every IPs are allow, or every IPs are disallow?

Reply:
Leaving it empty is not an option than the policy format gets messed up, i dont think you can do a null check on a helm value , so maybe with a boolean. `datadog.networkPolicy.cilium.ntp.enabled` and `datadog.networkPolicy.cilium.ntp.toCIDR`